assets/js/blocks/attribute-filter/style.scss

```diff
@@ -67,3 +67,10 @@
 		}
 	}
 }
+
+.editor-styles-wrapper  .wc-block-attribute-filter__button.wc-block-attribute-filter__button,
+.wc-block-attribute-filter__button.wc-block-attribute-filter__button {
+	padding: $gap-smaller $gap;
+	font-size: 0.875em;
+	border-radius: 2px;
```

Comment:
Do we really need to define the border-radius? Ideally we should let the theme control it.